examples/plot_virtual_evoked.py

```diff
@@ -0,0 +1,35 @@
+"""
+=======================
+Remap MEG channel types
+=======================
+
+In this example, MEG data are remapped from one
+channel type to another. This is useful for combining statistics
+for magnetometer and gradiometers.
```

Comment:
This is useful to:

- visualize combined magnetometers and gradiometers as magnetometers or gradiometers
- run statistics from both magnetometers and gradiometers while working with a single type of channels